app/presenters/document_export_presenter.rb

```diff
@@ -12,60 +12,127 @@ def as_json
+          last_author: stripped_last_author(edition),
+          topical_events: strippped_topical_events(edition),
+          translations: complete_translations(edition),
+          specialist_sectors: stripped_specialist_sectors(edition),
+        },
+      }.merge(provide_doctype_information(edition))
     end
   end
 
-  def edition_associations(edition)
-    output = {
-               edition: edition,
-               associations: {},
-             }
-
-    associations = edition.class.reflect_on_all_associations.map(&:name)
-    associations.each do |association|
-      if association == :images
-        edition_images = edition.public_send(association)
-        output[:associations][association] = complete_images_hash(edition_images)
-      elsif association == :attachments
-        edition_attachments = edition.public_send(association)
-        output[:associations][association] = complete_attachments_hash(edition_attachments)
-      else
-        output[:associations][association] = edition.public_send(association)
-      end
+  def stripped_last_author(edition)
+    author = edition.try(:last_author)
+    return {} unless author
+
+    { id: author.id, uid: author.uid }
+  end
+
+  def strippped_topical_events(edition)
+    topical_events = edition.try(:topical_events)
+    return [] unless topical_events
+
+    topical_events.map do |te|
+      { id: te.id, content_id: te.content_id }
     end
+  end
 
-    provide_doctype_information(edition, output)
-    output[:government] = edition.government
-    output[:whitehall_admin_links] = resolve_whitehall_admin_links(edition.body)
-    if edition.withdrawn?
-      output[:whitehall_admin_links].concat(resolve_whitehall_admin_links(edition.unpublishing.explanation))
+  def complete_translations(edition)
+    translations = edition.try(:translations)
+    return [] unless translations
+
+    translations.map do |translation|
+      translation.attributes.except("edition_id").merge(
+        "public_base_path" => Whitehall.url_maker.public_document_path(edition, locale: translation.locale),
+      )
+    end
+  end
+
+  def stripped_edition_policies(edition)
+    edition_policies = edition.try(:edition_policies)
+    return [] unless edition_policies
+
+    edition_policies.map do |e_policy|
+      { id: e_policy.id, policy_content_id: e_policy.policy_content_id }
+    end
+  end
+
+  def complete_organisations(edition)
+    edition_organisations = edition.try(:edition_organisations)
+    return [] unless edition_organisations
+
+    edition_organisations.map do |e_org|
+      org = e_org.organisation
+      { id: e_org.id, content_id: org.content_id, lead: e_org.lead, lead_ordering: e_org.lead_ordering }
     end
-    output
   end
 
-  def complete_images_hash(edition_images)
-    edition_images.map do |image|
+  def stripped_authors(edition)
+    authors = edition.try(:authors)
+    return [] unless authors
+
+    authors.map do |a|
+      { id: a.id, uid: a.uid }
```

Comment:
General suggestion for all these methods: avoid abbreviating the variables. This is longer but scans more easily (less thinking involved):

```suggestion
    authors.map do |author|
      { id: author.id, uid: author.uid }
```

Reply:
Sure, i'll un-abbreviate (don't think thats a word) all the things.